src/query_file_get_info.rs

```diff
@@ -0,0 +1,58 @@
+use hedera::{
+    query::{Query, QueryFileGetInfo, QueryFileGetInfoResponse},
+    Client, FileId,
+};
+use super::timestamp::py_date_time;
+use pyo3::prelude::*;
+use pyo3::types::PyDateTime;
+use crate::errors::PyException;
+
+#[pyclass(name = QueryFileGetInfo)]
+pub struct PyQueryFileGetInfo {
+    inner: Query<QueryFileGetInfoResponse>,
+}
+
+impl PyQueryFileGetInfo {
+    pub(crate) fn new(client: &Client, file: FileId) -> Self {
+        Self {
+            inner: QueryFileGetInfo::new(client, file),
+        }
+    }
+}
+
+#[pymethods]
+impl PyQueryFileGetInfo {
+    #[getter]
+    pub fn file_id(&mut self) -> PyResult<String> {
+        let response = self.inner.get().map_err(PyException)?;
```

Comment:
This should have a `.get` method that returns a `PyFileInfo` struct with all these getters

Look at the query I made